src/vscode-bicep/src/test/unit/surveys.unit.test.ts

```diff
@@ -131,7 +131,7 @@ describe("surveys-unittests", () => {
 
     // Show and respond with yes
     mocks.showInformationMessageMock.mockResolvedValueOnce(<MessageItem>{
-      title: "Sure",
+      title: "Jawohl",
```

Comment:
Shouldn't this be - Yes?

Reply:
Nope, title may be localized, it shouldn't matter what it is